internal/hostservice/service.go

```diff
@@ -0,0 +1,41 @@
+package hostservice
+
+import (
+	"github.com/blakewilliams/remote-development-manager/internal/hostservice/clipboard"
+	"github.com/blakewilliams/remote-development-manager/internal/hostservice/open"
+)
+
+// Service is the set of functionalities made available by the server to the client.
+type Service interface {
```

Comment:
```suggestion
type Runner interface {
```

Feel free to go with your opinion on this one, but thoughts on naming this `Runner` since it's responsible for running commands on the host OS?

One of the reasons I thought of another name was to reduce the stutter between `hostservice.Service`, but no big deal either way.